src/astrobleme/gui/Main.java

```diff
@@ -62,28 +62,56 @@ public void start(Stage primaryStage) throws Exception {
         canvas.setOnKeyPressed(e -> {
             Snake snake = grid.getSnake();
             if (loop.isKeyPressed()) {
-                return;
+                if(loop.isRunning()){
+                    System.out.println("Pressed ");
+                    return;
+                }
+                else{
+                }
```

Comment:
Please use <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>L</kbd> in IntelliJ (or android studio) to format the code automatically.

Also, it might be better to use a logger instead of `System.out.println()`. That way, we can set the log level and turn off unnecessary output when not necessary, like for the end user.

Reply:
I will try all suggestion promoted in this page.